service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -267,6 +270,11 @@ private List<RunStateResponse> buildInputsAndSubmitRun(
         log.warn(errorMsg, e);
         runStateResponseList.add(
             storeRun(runId, null, runSet, record.getId(), SYSTEM_ERROR, errorMsg + e.getMessage()));
+      } catch (WorkflowAttributesNotFoundException e) {
+        String errorMsg = "Attribute was not found in WDS record";
+        log.warn(errorMsg, e);
+        runStateResponseList.add(
+            storeRun(runId, null, runSet, record.getId(), SYSTEM_ERROR, e.getMessage()));
```

Comment:
do we want to prepend `errorMsg` to `e.getMessage()` like we do in other catch blocks?

Reply:
This what I had initially, but that caused the two of the same error message to be printed in the DB. So in this case, I just opted for `e.getMessage`. If you think I should re-add `errorMsg` I'd be happy to.

Reply:
ah, got it. no need in that case